src/pages/Task.tsx

```diff
@@ -129,6 +140,28 @@ const TaskCore: React.FC = () => {
       taskAnalytics.sendEvent({ name: "Change Tab", tab }),
   });
 
+  // Query build baron data
+  const { data: buildBaronData, error: buildBaronError } = useQuery<
+    BuildBaronQuery,
+    BuildBaronQueryVariables
+  >(GET_BUILD_BARON, {
+    variables: { taskId: id, execution },
+    pollInterval,
+  });
+  const buildBaron = buildBaronData?.buildBaron;
+  const buildBaronConfigured = buildBaron?.buildBaronConfigured;
+
+  // logic for displaying the build baron tab
+  const timeZone = useUserTimeZone();
```

Comment:
we don't need this variable since the date-fns format function can handle timezone differences as long as the time in UTC (i think it is)